.gitmodules

```diff
@@ -32,3 +32,8 @@
 [submodule "suru-plus"]
 	path = plugins/icons_suruplus/suru-plus
 	url = https://github.com/gusbemacbe/suru-plus.git
+[submodule "suru-plus-aspromauros"]
+	path = plugins/icons_suruplus_aspromauros/suru-plus-aspromauros
+	url = https://github.com/gusbemacbe/suru-plus-aspromauros.git
+
+
```

Comment:
please remove empty space

Reply:
Ready! If it goes something, I will rebase and then git pull upstream.

I understood how the rebase works. Rebase is like downgrade and is like as if the starship USS Enterprise travelled through the alternate timeline.